packages/react-langgraph/src/useLangGraphRuntime.ts

```
@@ -55,7 +54,7 @@ const getMessageContent = (msg: AppendMessage) => {
         throw new Error("Tool call appends are not supported.");
 
       default:
-        const _exhaustiveCheck: "file" | "audio" = type;
+        const _exhaustiveCheck: "reasoning" | "source" | "file" | "audio" = type;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)